### ISSUE TITLE
About Page

### ISSUE THREAD

**Issue by suorcd** [PodcastGenerator/PodcastGenerator#676]
**Description**
I would like to see an link to the an About page under the 'Podcast Generator' tab.
I would be nice to see the current version, License, and links to help documentation.
Maybe in the future add a version check and app 'health check'.

**Motivation**
While trying to figure out if i had already upgraded the app, I noticed that this information wasn't easily found.
I think this is distinct from the Admin Home tab that has general Podcast Generator News.

**Legal aspect**

* [ X] I am the inventor of this idea or the idea is public domain
* [ X] I release this idea into the public domain

**Comment by coldacid** [comment]
Good idea. I'll try to squeeze it in for 3.3 but right now I'm very much focused on trying to get live items and value tags in place.

**Comment by ushbyorg** [comment]
Do you mean like this,

![122](https://user-images.githubusercontent.com/85970369/183898663-9975ad12-14f4-4bcb-97d5-87cd40707691.png)

I can do that and configure it with all the script and perhaps allow you to modify the page through admin but, Legal aspect as in to whom and where ? can you clarify ? this is open source and free!

**Comment by suorcd** [comment]
For the license side, I was thinking the title and a link to the license of the project.
When thinking about the request, I was looking at the 'About Mozilla Firefox', 
![image](https://user-images.githubusercontent.com/60907848/184463349-b986b311-0f29-467b-9d53-a733959f913b.png)

and I was wondering about the license of the project after I installed it.
I might be the only person who wants this, but I think it would be useful to have.
